chaos/cluster.go

```diff
@@ -0,0 +1,258 @@
+package chaos
```

Comment:
I guess I'm also confused about what is particularly chaotic about this package.